<launch>
	<arg name="Marker_duration" default="0.1" />
	<arg name="Voxel_leafsize" default="0.3" />
	<arg name="threshold_range" default="10.0" />

	<node pkg="perception" type="perception" name="lidar_detection" args="input:=/velodyne_points" output="screen">
		<param name="Marker_duration" value="$(arg Marker_duration)"/>
		<param name="Voxel_leafsize" value="$(arg Voxel_leafsize)"/>
		<param name="threshold_range" value="$(arg threshold_range)"/>
	</node>
</launch>
